apps/marketplace/components/Opportunity/Opportunity.js

```diff
@@ -216,7 +223,9 @@ const Opportunity = props => {
                 <strong>Estimated start date</strong>
               </div>
               <div className="col-xs-12 col-sm-8">
-                {brief.lotSlug === 'specialist' ? format(brief.startDate, 'D-MM-YYYY') : brief.startDate}
+                {brief.lotSlug === 'specialist' && brief.startDate
```

Comment:
doesn't this still display it?

Reply:
This is to prevent `Invalid Date` from showing.

![Screen Shot 2019-09-05 at 9 49 40 am](https://user-images.githubusercontent.com/2645932/64301298-81374980-cfc2-11e9-9651-9008bf131f30.png)

Reply:
I think the default values for non-specialist were ok but I've moved this logic into a function.